app/models/concerns/exportable.rb

```diff
@@ -0,0 +1,15 @@
+module Exportable
```

Comment:
I thought you might say that. Now to figure out a light way to spec the Exportable concern given as_csv's Demeter violations and assumptions. Setting up a sham AR::Base class may get tricky. I'll research.